src/index.tsx

```diff
@@ -454,6 +497,57 @@ class App extends React.Component<{}, AppState> {
   public render() {
     return (
       <>
+        <div className="wrappers">
+          <div className="saveWrapper">
+            <button disabled={elements.length === 0} onClick={save}>
+              Save
```

Comment:
I think that an explicit save button isn't a good idea in practice here. I feel like we should just always save to localstorage and restore it when it comes back